aif360/sklearn/metrics/metrics.py

```diff
@@ -1,14 +1,16 @@
 import warnings
 
 import numpy as np
+import pandas as pd
 from sklearn.metrics import make_scorer as _make_scorer, recall_score
 from sklearn.metrics import multilabel_confusion_matrix
 from sklearn.neighbors import NearestNeighbors
 from sklearn.utils import check_X_y
 from sklearn.exceptions import UndefinedMetricWarning
 
 from aif360.sklearn.utils import check_groups
-
+from aif360.metrics.mdss.ScoringFunctions import Bernoulli
```

Comment:
Why only Bernoulii is imported? What about Berk Jones and Poisson? Are they not incorporated yet?

Reply:
The original paper on Bias Scan of classifiers used Bernoulli. For a straightforward application of MDSS as a classification metric Bernoulli makes the most sense. Advanced users will be able to use other scoring functions as they please.